src/GameServer/World/Generics/NpcBypasses/SellJunk.js

```
@@ -1,13 +1,16 @@
 const ServerResponse = invoke('GameServer/Network/Response');
 const Database       = invoke('Database');
 const ItemDisposition = invoke('GameServer/Bot/Economy/ItemDisposition');
+const ShotStock = invoke('GameServer/Inventory/ShotStock');
 
 module.exports = function(session, parts) {
```

ESLint

[error] 'parts' is defined but never used.

(no-unused-vars)